doc/manual/manual.tex

```diff
@@ -10527,16 +10528,31 @@ \subsubsection{Section in the manual}
   \end{verbatim}
   to insert the code in the manual.tex file.
 \item Explain what the important input parameters in this setup are, what values you
-  set them to and why. Ideally, this description includes snippets from
+  set them to and why. This paragraph should give an overview over your model setup, 
+  including the initial conditions, boundary conditions, geometry, etc. and anything that 
+  is special about the setup. Ideally, this description includes snippets from
   the input file. You can place these snippets in the subfolder you created in the
   \href{doc/manual/cookbooks/.}{doc/manual/cookbooks}
   directory and include them in the manual.tex file using a command like
   \begin{verbatim}
-  \lstinputlisting[language=prmfile]{cookbooks/subfolder_name/input_snippet.prm}!
+  \lstinputlisting[language=prmfile]{cookbooks/subfolder_name/input_snippet.prm}
   \end{verbatim}
 \item Show the model results in form of figures and/or plots, accompanied by an explanation
   of what happens in the model. This can also include a link to an animation of the model
   you made and uploaded somewhere, for example on youtube.
+  When creating figures or animations, you should think about the color scale that you use. 
+  Some colormaps -- like the rainbow color palette that is still the default in some 
+  visualization tools -- can obscure features present in the data and introduce 
+  artifacts, because the rainbow color scale is not preceptually uniform. For more background on this topic, 
+  there is a great summary on \href{https://matplotlib.org/users/colormaps.html}{matplotlib}. 
```

Comment:
just use `\url` here -- it's worth showing the whole URL here, whereas "great summary on matplotlib" seems like grammatically incorrect